src/test/java/io/stargate/sgv2/jsonapi/api/configuration/ObjectMapperConfigurationTest.java

```diff
@@ -42,6 +46,101 @@ class ObjectMapperConfigurationTest {
 
   @Inject DocumentLimitsConfig documentLimitsConfig;
 
+  @Nested
+  class unmatchedOperationCommandHandlerTest {
+    @Test
+    public void customHandlerTest() throws Exception {
+      String json =
+          """
+                  {
+                    "findOne": {
+                    }
+                  }
+                  """;
+      final ObjectMapper objectMapper = new ObjectMapper();
+      objectMapper.addHandler(
+          new DeserializationProblemHandler() {
```

Comment:
If refactored in `ObjectMapperConfiguration`, could and should reuse implementation here (to avoid code duplication).

Reply:
This is mostly for testing if the custom handler worked as a methodology.
I did extract a class to extend the DeserializationProblemHandler though.
Thanks!